ansible/roles/machine/provision_ipaserver/tasks/main.yml

```diff
@@ -5,6 +5,102 @@
     dest: /usr/local/bin/ipa-run-webui-tests
     mode: 0755
 
+- name: fetch hostname
+  command: hostname -f
+  register: master_hostname
+  when: caless == true
```

Comment:
The hostname is set earlier to` {{ inventory_hostname }}.ipa.test` (in https://github.com/freeipa/freeipa-pr-ci/blob/master/ansible/roles/machine/provision/tasks/main.yml#L95). You can remove this task and use `{{ inventory_hostname }}.ipa.test` instead of `{{ master_hostname }}` later on.